### ISSUE TITLE
I want a rule that will find tests without assert statements

### ISSUE THREAD

**Issue by AlexTereshenkov** [m-burst/flake8-pytest-style#204]
# Rule request

## Description

Find tests that doesn't have any `assert` statements.

Bad:

```
def test_func():
  db.run('insert rows')
```

Good:

```
def test_func():
  db.run('insert rows')
  assert db.select('*') > 0
```

## Rationale

In most cases, a test without an `assert` statement is not really a test. It's likely that a developer has forgotten to include an `assert` statement after refactoring. Such tests would pass without actually checking the output of exercising a piece of code.

**Comment by m-burst** [comment]
Hi @AlexTereshenkov,

This rule, if implemented as you are suggesting, will trigger a lot of false positives, for example:
* if the asserts are extracted into a helper function to avoid repetition between multiple tests
* if the test is checking that is exception is being raised using a `pytest.raises` block

The first scenario with the helper functions is too complicated to handle on the AST level, so I do not find it reasonable to implement.
